### PR TITLE
migrate from python2 to python3 of rst2man.py

### PR DESCRIPTION
**Pull request by yetist**
I use the follow command to test:
```
$ python2 ./rst2man.py caja-dropbox.txt > 2.1
$ python3 ./rst2man.py caja-dropbox.txt > 3.1
$ diff -u 2.1 3.1 
```
Please confirm if this is good.

### CODE REVIEW

File: rst2man.py [hunk 1]
```diff
@@ -1081,6 +1080,7 @@ def depart_warning(self, node):
         self.depart_admonition()
 
     def unimplemented_visit(self, node):
+        raise NotImplementedError(node.astext())
```
Comment (monsta) [inline rst2man.py]: Is this needed? There's already an exception raised on the next line...

**Comment by monsta** [comment]
The test works. I would also change `python` to `python3` on the first line of the file.

File: rst2man.py (hunk 1 above)
Comment (yetist) [reply rst2man.py]: My mistake, this line can be removed.

**Comment by monsta** [comment]
I wonder if we need this in `configure.ac` when the system script is used?
```
PYTHON_CHECK_MODULE(docutils, docutils)
```
For the local script it has to be changed to `PYTHON3_CHECK_MODULE`.

**Comment by monsta** [comment]
@raveit65: BTW, how do you build it in Fedora 30? Currently `configure.ac` checks for Python 2 version of docutils...

**Comment by raveit65** [comment]
@monsta 
For caja/nemo/nautilus-dropbox build at rpm rpmfusion i didn't update to latest nautilus-dropbox code from master.
Last build of n-d use only this commit from upstream https://pkgs.rpmfusion.org/cgit/nonfree/nautilus-dropbox.git/commit/
For caja-dropbox i use only the caja-extension part.
https://pkgs.rpmfusion.org/cgit/nonfree/caja-dropbox.git/tree/caja-dropbox.spec#n51

Well, we are waiting that dropbox upstream  fixes last remaining issue.
Fedora 30-mass-rebuild is done, and f30 will be branched these days from rawhide, so we lost python2.
Rpmfusion isn't official fedora, but same maintainers ;)
So, we don't follow fedora's schedule strictly, mostly there is a delay......

**Comment by monsta** [comment]
So you still had Python 2 (and `python-docutils`) in rawhide until now?

**Comment by raveit65** [comment]
No, i noticed that a pluma test build with python2  stuff failed since today.
I only said , that we still ignore the problem and wait for nautilus-dropbox developer action.
If they don't fix it, me and Leigh123Linux will drop dropbox from rpmfusion.

**Comment by monsta** [comment]
Ok, so caja-dropbox build might fail then - even if you use only the extension library, it's still needed to run `./configure`.

**Comment by raveit65** [comment]
Can this be merged?

**Comment by yetist** [comment]
I can't use dropbox, so I can't confirm if the dropbox is working.
Sorry, I can't give any advice. It is best to ask other people.

**Comment by monsta** [comment]
I've added one missing change and squashed + merged it. The build still needs Python 2 version of docutils, I'll fix that.

**Comment by raveit65** [comment]
Btw. new nautilus-dropbox is released. See what Leigh did at rpmfusion.
http://pkgs.rpmfusion.org/cgit/nonfree/nautilus-dropbox.git/diff/python3-docutils_fix.patch

**Comment by monsta** [comment]
He can also remove `AC_PATH_PROG([PYTHON], [python])` which checks for Python 2 binary...

**Comment by monsta** [comment]
Ok, I hope https://github.com/mate-desktop/caja-dropbox/pull/34 does it.

**Comment by raveit65** [comment]
Which PR we need?
This one? or the other one? or both?

**Comment by monsta** [comment]
Well, this one is merged by me, we need it :slightly_smiling_face: 
The other one completes migration to Python 3.

**Pull request merged**